framework-plugins/lisk-framework-http-api-plugin/src/controllers/delegates.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { Request, Response, NextFunction } from 'express';
+import { validator, LiskValidationError } from '@liskhq/lisk-validator';
+import { BaseChannel, PluginCodec } from 'lisk-framework';
+import { paginateList } from '../utils';
+
+const getDelegatesQuerySchema = {
+	type: 'object',
+	properties: {
+		limit: {
+			type: 'string',
+			format: 'uint32',
+			description: 'Number of delegates to be returned',
+		},
+		offset: {
+			type: 'string',
+			format: 'uint32',
```

Comment:
if uint32, type: 'integer' is fine here, and you could add min and max

Reply:
when `const errors = validator.validate(getDelegatesQuerySchema, req.query);` it will throw error for integers as everything in query is of string type